File: app/src/main/java/collection/ex01.java
package collection;

import java.util.ArrayList;
import java.util.List;

public class ex01 {
    public static void main(String[] args) {
        List<String> names = new ArrayList<>();
        names.add("Alice");
        names.add("Bob");
        names.add("Charlie");

        names.remove("Bob");

        System.out.println("contains Alice: " + names.contains("Alice"));
        System.out.println("list: " + names);
    }
}

File: app-kotlin/src/main/kotlin/collection/ex01.kt
package collection

class ex01 {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val names = mutableListOf("Alice", "Bob", "Charlie")
            names.remove("Bob")
            println("contains Alice: ${names.contains("Alice")}")
            println("list: $names")
        }
    }
}
